Userland/Libraries/LibJS/Runtime/WeakContainer.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021, Idan Horowitz <idan.horowitz@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Heap/Heap.h>
+
+namespace JS {
+
+class Heap;
```

Comment:
We already include `Heap.h`, why is this needed?

Reply:
this was left over from testing, removed.